Apps/GatewayApi/src/Models/UserProfileModel.cs

```diff
@@ -123,6 +123,7 @@ public static UserProfileModel CreateFromDbModel(UserProfile model)
                 IsSMSNumberVerified = !string.IsNullOrEmpty(model.SMSNumber),
                 LastLoginDateTime = model.LastLoginDateTime,
                 ClosedDateTime = model.ClosedDateTime,
+                AcceptedTermsOfService = model.TermsOfServiceId != Guid.Empty,
```

Comment:
This value needs to reflect whether the model.TermsOfServiceId matches the most recent Terms of Service.

Reply:
Scratch that - this is captured in the HasTermsOfServiceUpdated property.